### PR TITLE
Support Zeitwerk 2.7.0+

### PR DESCRIPTION
**Pull request by DawidJanczak**
Changed in 2.7.0, see https://github.com/fxn/zeitwerk/commit/8f37a65ee5c18f0fa3e1bf28283e8ee4b1b9b39e

Close https://github.com/sinatra/sinatra/issues/2047

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -51,7 +51,7 @@ gem 'redcarpet', platforms: [:ruby]
 gem 'simplecov', require: false
 gem 'slim', '~> 5'
 gem 'yajl-ruby', platforms: [:ruby]
-gem 'zeitwerk', '< 2.7.0' # https://github.com/sinatra/sinatra/issues/2047
+gem 'zeitwerk'
```
Comment (dentarg) [inline Gemfile:54]: We should probably try with the old version too, in CI. Hopefully I'll have some time next week if you need help with that.

Comment (DawidJanczak) [reply Gemfile]: Just pushed that, can you have a look if that's what you had in mind or if you'd rather do it differently?

Comment (dentarg) [reply Gemfile]: Looks good to me :) I hope to merge this next week

**Comment by DawidJanczak** [comment]
Thanks for merging this! Are there plans to make a release containing this fix?

**Comment by dentarg** [comment]
Yes, this week

**Pull request merged**